libsoftwarecontainer/src/gateway/devicenode/devicenodegateway.cpp

```diff
@@ -83,6 +93,8 @@ ReturnCode DeviceNodeGateway::applySettings()
             if (job.wait() != 0) {
```

Comment:
Would be nice to have a better, or maybe more verbose, way of determine if a job is a failure or not. 
Since all of the code that is run inside a container now use either FunctionJob or CommanJob, I think we should invest in some convenience in these classes. For instance, isSuccess() or isError() would be nice.

Might be out of scope for the pull request though...